src/service/history-impl.js

```diff
@@ -0,0 +1,659 @@
+/**
```

Comment:
Weird. Is it not catching the "move" op? Would be good to preserve the revisions...